### DISCUSSION TITLE
What are the factors that affect the proportion of feces excreted?

### DISCUSSION THREAD

**Discussion by SCZ1174** [Open-Systems-Pharmacology/Forum#1688] [Questions & Problems]
Hello everyone,
I'm building a model by fitting different clearance pathway ratios and I'm having a little problem.
The fecal excretion ratio of my drug is 13%. I used the following method to try to fit it, but I don’t know if it is suitable:

1. Add bile clearance. (I saw an increase in bile clearance ratio, but no change in fecal clearance ratio.)
2. Add the enterohepatic circulation process on the basis of 1. (Added "Galbladder emptying enabled" to the event module and set "EHC continuous fraction" to 0.5. It was found that the bile clearance ratio decreased, but the feces clearance ratio remained.)
3. Delete the enterohepatic circulation process, assume an efflux secretion protein and set its expression in each intestinal segment, and the fecal clearance ratio changes!

- Why does the proportion of feces excretion remain unchanged after bile is added for elimination?
- After adding the enterohepatic circulation process, the reason why the fecal excretion ratio remains unchanged is that the drug is absorbed again? And why does the bile clearance ratio decrease?
- Do I have to add secreted proteins from the gut if I want to fit the fecal excretion ratio?

**Comment by PavelBal** [comment]
To achieve fecal excretion, you have to limit intestinal absorption of the compound in some way. This can be done either by limiting the solubility or the intestinal permeability of the compound. Only if the absorption is limited in some way, you will get an increase of fraction excreted to feces after introducing biliary clearance.

Limiting intestinal permeability can be done by directly reducing the parameter "Specific intestinal permeability" or introducing efflux transporters like P-gp.
